app/services/foreman_gutterball/gutterball_service.rb

```diff
@@ -26,26 +26,12 @@ class GutterballService < GutterballResource
     def initialize(_options = {})
     end
 
-    def available_reports
+    def reports(report_key = nil, query_params = nil)
```

Comment:
would being more specific about params mean more changes for foreman_gutterball if the gb api is in flux?